src/components/Editor/ContentContainer.vue

```diff
@@ -67,6 +67,12 @@ export default {
 		width: 100%;
 	}
 
+	@media (max-width: 670px) {
```

Comment:
It wasn't immediately obvious to me that this is the value of `--text-editor-max-width`, maybe add a code comment to this.

Maybe we can just set a max-width in the above general `.editor_content` rule like this to keep it simple?

```css
.editor__content {
	max-width: min(var(--text-editor-max-width, calc(100vw - 16px));
```

Reply:
Ok, please check again.